dpdata/plugins/rdkit.py

```diff
@@ -0,0 +1,43 @@
+from dpdata.format import Format
+try:
+    import rdkit.Chem
+except ModuleNotFoundError:
+    pass
+import dpdata.rdkit.utils
+
+
+@Format.register("mol")
+@Format.register("mol_file")
```

Comment:
Why here are 2 register decorators?

Reply:
All of the following usage will be supported:
* `from_mol(file_name)`
* `from_mol_file(file_name)`
* `to_mol(file_name)`
* `to_mol_file(file_name)`
* `BondOrderSystem(file_name, fmt='mol')`
* `BondOrderSystem(file_name, fmt='mol_file')`
* `to('mol', file_name)`
* `to('mol_file', file_name)`